File: Onoicrm.Domain/Entities/Tooth.cs
namespace Onoicrm.Domain.Entities;

public class Tooth : Entity
{
    public string Caption { get; set; } = "";
    public string Description { get; set; } = "";
    public int Position { get; set; }
    public int Quarter { get; set; }
    public ICollection<Channel> Channels { get; set; } = new List<Channel>();
    protected override int GetClassId() => ClassNames.Tooth;
}

File: Onoicrm.Admin/src/entities/Tooth.ts
import { Entity } from './Entity';
import { Clinic } from './Clinic';

export class Tooth extends Entity {
  public caption!: string;
  public description!: string;
  public position!: number;

  constructor(instance?: Partial<Clinic>) {
    super();
    Object.assign(this, instance);
  }
}